integration/js/utils/SdkBaseTest.js

```diff
@@ -35,15 +35,17 @@ class SdkBaseTest extends KiteBaseTest {
     this.testReady = false;
     this.testFinish = false;
     this.testName = testName;
-    this.useSimulcast = !!this.payload.useSimulcast;
+    this.useSimulcast = this.payload?.useSimulcast || false;
```

Comment:
What is the difference here ? !!this.payload.useSimulcast seems shorter.

Reply:
Mostly impossible, but if there's no `payload` in the `test.config.json` file, the original one will throw an exception. We could also use `!!this.payload?.Simulcast`, but the current one is more specific to me cause it explicitly show the default value of `useSimulcast`.

Reply:
I will use `!!this.payload?.Simulcast` instead.